src/Editor.js

```diff
@@ -0,0 +1,127 @@
+import * as THREE from 'three';
+import { getCurrentTime } from './Util.js';
+
+export class Editor {
+  constructor(viewer, options = {}) {
+    this.viewer = viewer;
+
+    this.editPanel = null;
+
+    this.pointerUpHandlerPlaneFinder = this.onPointerUpPlaneFinder.bind(this);
+
+    this.initialized = false;
+    this.init();
+  }
+
+  init() {
+    if (this.initialized) return;
+
+    this.setupEditPanel();
+
+    this.initialized = true;
+  }
+
+  setupEditPanel() {
+    this.editPanel = document.createElement('div');
+    this.editPanel.style.position = 'absolute';
+    this.editPanel.style.padding = '10px';
+    this.editPanel.style.backgroundColor = '#cccccc';
+    this.editPanel.style.border = '#aaaaaa 1px solid';
+    this.editPanel.style.zIndex = 90;
+    this.editPanel.style.width = '375px';
+    this.editPanel.style.fontFamily = 'arial';
+    this.editPanel.style.fontSize = '10pt';
+    this.editPanel.style.textAlign = 'left';
+
+
+    const editTable = document.createElement('div');
+    editTable.style.width = '100%';
+
+
+    // Plane Finder
+    const planeFinder = document.createElement('div');
+    planeFinder.style.width = '100%';
+    planeFinder.style.display = 'flex';
+    planeFinder.style.flexDirection = 'row';
+    planeFinder.style.justifyContent = 'space-between';
+
+    const planeFinderLabel = document.createElement('p');
+    planeFinderLabel.id = 'planeFinderLabel';
+    planeFinderLabel.innerHTML =
```

Comment:
I see this code in multiple places, maybe it's worth making an `updateCameraUpLabel()` function (or something like that).

Reply:
Yes, there are a few places where we could refactor to separate functions